docs/src/content/docs/id/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,287 @@
+    				// Path ke Tailwind base styles Anda:
+    				'./src/tailwind.css',
+    			],
+    		}),
+    		tailwind({
+    			// Nonaktifkan base styles bawaan
+    			applyBaseStyles: false,
+    		}),
+    	],
+    });
+    ```
+
+5.  Tambahkan plugin Tailwind Starlight ke `tailwind.config.cjs`:
+
+    ```js ins={2,7}
+    // tailwind.config.cjs
+    const starlightPlugin = require('@astrojs/starlight-tailwind');
+
+    /** @type {import('tailwindcss').Config} */
+    module.exports = {
+    	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+    	plugins: [starlightPlugin()],
+    };
+    ```
+
+### Styling Starlight dengan Tailwind
+
+Starlight akan menggunakan nilai-nilai dari konfigurasi tema [Tailwind Anda](https://tailwindcss.com/docs/theme) di dalam UI-nya.
+
+Jika diatur, opsi berikut akan menggantikan style bawaan Starlight:
+
+- `colors.accent` — digunakan untuk tautan dan menandakan item yang sedang aktif
+- `colors.gray` — digunakan untuk warna latar belakang dan border
+- `fontFamily.sans` — digunakan untuk UI dan konten teks
+- `fontFamily.mono` — digunakan untuk contoh kode
+
+```js {12,14,18,20}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Warna aksen pilihan Anda. Indigo paling mendekati bawaan Starlight.
+				accent: colors.indigo,
+				// Skala abu-abu pilihan Anda. Zinc paling mendekati bawaan Starlight.
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				// Font teks pilihan Anda. Starlight menggunakan kumpulan font sistem secara default.
+				sans: ['"Atkinson Hyperlegible"'],
+				// Font kode pilihan Anda. Starlight menggunakan font monospace sistem secara default.
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Tema
+
+Tema warna Starlight dapat dikontrol dengan mengganti custom properti bawaannya.
+Variabel-variabel ini digunakan di seluruh UI dengan berbagai nuansa abu-abu yang digunakan untuk warna teks dan latar belakang, dan satu warna aksen yang digunakan untuk tautan serta menyoroti item-item yand sedang aktif di bagian navigasi.
+
+### Editor Tema Warna
+
+Gunakan slider di bawah ini untuk mengubah palet warna aksen dan abu-abu Starlight.
+Preview gelap dan terang akan menampilkan warna-warna hasilnya, dan seluruh halaman juga akan diperbarui untuk menyesuaikan dengan perubahan yang Anda lakukan.
+
+Saat Anda puas dengan perubahan Anda, salin CSS atau kode Tailwind di bawah ini dan gunakan di proyek Anda.
+
+import ThemeDesigner from '../../../../components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Presets',
+			ocean: 'Ocean',
+			forest: 'Forest',
+			oxide: 'Oxide',
+			nebula: 'Nebula',
+			default: 'Default',
+			random: 'Random',
```

Comment:
Noticing that these haven’t been translated. Was that intentional? In other languages we generally have translated these.

Reply:
Thanks for catching that, I missed those